content/sensu-go/5.0/guides/prometheus-metrics.md

```diff
@@ -0,0 +1,455 @@
+---
+title: "Using Prometheus with the Sensu Prometheus Collector"
+linkTitle: "Using Prometheus with Sensu"
+weight: 120
+version: "5.0"
+product: "Sensu Go"
+platformContent: false
+menu: 
+  sensu-go-5.0:
+    parent: guides
+---
+
+- [What is the Sensu Prometheus Collector?](#what-is-the-sensu-prometheus-collector)
+- [Install and Configure Prometheus](#install-and-configure-prometheus)
+- [Install and Configure Sensu Go](#install-and-configure-sensu-go)
+- [Install Sensu Prometheus Collector](#install-sensu-prometheus-collector)
+- [Install Sensu InfluxDB Handler](#install-sensu-influxdb-handler)
+- [Install and Configure InfluxDB](#install-and-configure-influxdb)
+- [Install and Configure Grafana](#install-and-configure-grafana)
+
+## What is the Sensu Prometheus Collector?
+
+The [Sensu Prometheus Collector][1] is a sensu check plugin that can collect metrics from a [Prometheus exporter][2] or the [Prometheus query API][3]. The allows Sensu to then send metrics to one or more time-series databases, such as InfluxDB or Graphite.
+
```

Comment:
Feels like we need a section to describe why I would want to do this.

Reply:
Agreed. As I attempt to write this though, I am finding myself just reiterating what the Prometheus Collector does vs an actual user use case.

Maybe we can pair up on Monday or during office hours to come up with something that is valuable to readers.